index.html

```diff
@@ -0,0 +1,124 @@
+
+<!--This is the navigation - current page has id of active for styling in navigation later-->
+    <ul>
+      <li class="navigation"><a id="active" href="index.html">Home /</a></li>
+      <li class="navigation"><a href="dogs.html">Dogs /</a></li>
+      <li class="navigation"><a href="blog.html">Blog</a></li>
+    </ul>
+
+<!-- This is the Adopt A Dog section -->
+    <section id="intro" class="grid-container-intro">
+      <img src="images/home-intro.jpg" alt="Picture of a boy hugging his dog.">
+      <h2>Adopt A Dog</h2>
+      <p>Dat tungg tho borkf many pats tungg adorable doggo very good spot long doggo, lotsa pats super chub long water shoob wrinkler. Pupperino clouds thicc heckin good boys such treat aqua doggo, lotsa pats boof adorable doggo very good spot. Waggy wags much ruin diet very good spot smol borking doggo with a long snoot for pats, adorable doggo borkf. Very taste wow the neighborhood pupper super chub you are doing me the shock borkdrive, wow such tempt stop it fren.</p>
+    </section>
+
+<!-- This is all the content for the grid of dogs - each element has a class for potential positioning later-->
+    <section id="dog-content" class="grid-container-dogs">
+      <article class="murphy">
+        <img class="murphy-img" src="images/murphy-card.jpg" alt="Image of Murphy who is a brown dog with white muzzle.">
+        <h3 class="murphy-name">Murphy</h3>
+        <p class="murphy-cost">Cost to Adopt: $123.45</p>
+        <p class="murphy-desc">Long doggo floofs very hand that feed shibe aqua doggo, noodle horse.</p>
+        <a class="murphy-adopt" href="index.html">Adopt</a>
+      </article>
+
+      <article class="poppy">
+        <img class="poppy-img" src="images/poppy-card.jpg" alt="Image of Poppy who is a white dog with brown ears.">
+        <h3 class="poppy-name">Poppy</h3>
+        <p class="poppy-cost">Cost to Adopt: $123.45</p>
+        <p class="poppy-desc">Long doggo floofs very hand that feed shibe aqua doggo, noodle horse.</p>
+        <a class="poppy-adopt" href="index.html">Adopt</a>
+      </article>
+
+      <article class="jack">
+        <img class="jack-img" src="images/jack-card.jpg" alt="Image of Jack who is a brown brown and white beagle.">
+        <h3 class="jack-name">Jack</h3>
+        <p class="jack-cost">Cost to Adopt: $123.45</p>
+        <p class="jack-desc">Long doggo floofs very hand that feed shibe aqua doggo, noodle horse.</p>
+        <a class="jack-adopt" href="index.html">Adopt</a>
+      </article>
+
+      <article class="duffy">
+        <img class="duffy-img" src="images/duffy-card.jpg" alt="Image of Duffy who is a brown dog with white face and chest.">
+        <h3 class="duffy-name">Duffy</h3>
+        <p class="duffy-cost">Cost to Adopt: $123.45</p>
+        <p class="duffy-desc">Long doggo floofs very hand that feed shibe aqua doggo, noodle horse.</p>
+        <a class="duffy-adopt" href="index.html">Adopt</a>
+      </article>
+
+      <article class="lucas">
```

Comment:
For all of these classes I'd use the same class for each article.  <article class="dog-tile">, <img class="dog-tile-image"> etc. You'll be able to style them all with one set of classes.

Reply:
So do you mean the same class for multiple articles? Like "class="dogs" for ALL the dog articles?